### PR TITLE
CLC-5601 Import first_name/last_name but compare to Canvas full_name

### PR DESCRIPTION
**Pull request by raydavis**
https://jira.ets.berkeley.edu/jira/browse/CLC-5601

Slight tweak to @paulkerschen 's last, based on looking at the full production user list.

### CODE REVIEW

File: fixtures/csv/Canvas_provisioning_report_users_csv.csv [hunk 1]
```diff
@@ -1,30 +1,5 @@
...
-24,300846,300846,Stu,Testb,test-300846@example.com,active
-25,322583,322583,Empstaff,Testc,test-322583@example.com,active
-31,300940,300940,Svetlana,Masterkova,svetlana@example.com,active
-46,300939,300939,Tammi,Chang,tammi.chang.clc@example.com,active
-63,11002820,11002820,Tammi,Chang,tammi.chang.clc@example.com,active
-64,61889,61889,Oski,Bear,oski.bear.clc@example.com,active
-65,730565,730565,Michelle,Someone,michelle@example.com,active
-72,999,canvassupport,Canvas,Support,canvassupport@example.com,active
+canvas_user_id,user_id,login_id,first_name,last_name,full_name,sortable_name,short_name,email,status
+canvas_user_id,user_id,login_id,first_name,last_name,full_name,sortable_name,short_name,email,status
```
Comment (johncrossman) [inline fixtures/csv/Canvas_provisioning_report_users_csv.csv]: @raydavis Two header rows. Not intentional?

**Comment by johncrossman** [comment]
Looks good. I'll wait for Ray's answer on header rows.

File: fixtures/csv/Canvas_provisioning_report_users_csv.csv (hunk 1 above)
Comment (raydavis) [inline fixtures/csv/Canvas_provisioning_report_users_csv.csv]: Def not! Thank you, @johncrossman .

**Comment by paulkerschen** [comment]
Good to go, I should say.

**Pull request merged**